panoramisk/fast_agi.py

```diff
@@ -46,16 +46,16 @@ def call_waiting(request):
         return agi_result
 
 
-class Application(dict):
```

Comment:
-1 for this name: you already know it's a fastagi application with the name of the file.